tests/end-to-end/logging/testdox-text.phpt

```diff
@@ -22,3 +22,4 @@ unlink($output);
 Bank Account (PHPUnit\TestFixture\BankAccount)
  [x] Balance is initially zero
  [x] Balance cannot become negative
+ [x] Balance cannot become negative
```

Comment:
This change in the test documents a change in behavior. I do not think that this change is desired and wonder how this is related to the topic of this PR?

Reply:
I had written a description in the previous commit before it was squashed to comply with your request.

The reason there is 2 of them is because the test has two test methods that look identical: https://github.com/sebastianbergmann/phpunit/blob/ccb711a9b71d02741712961bdd1d7e519df9e460/tests/_files/BankAccountTest.php#L48-L78

`testBalanceCannotBecomeNegative` and `testBalanceCannotBecomeNegative2`

but the previous implementation used the method names as unique keys, and since the "2"  was removed from `testBalanceCannotBecomeNegative2` to "normalize" the test method name, so it only appeared once.